pkgs/by-name/tp/tproxy/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```

Simpler diff in the future.

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:
```